submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
+  elem = document.createElement(elem);
+  elem.classList.add(className);
+  for (attrName in attrObj) {
+    elem.setAttribute(attrName, attrObj[attrName]);
+  }
+  return elem;
+};
+
+const createMenu = (data) => {
+  let menu = createElem("ul", { id: "menu", class: "menu" });
+  data.forEach(
+    (item, index) => { menu.appendChild(createMenuItem(data[index], index)) }
+  )
+  return menu;
+};
+
+const createMenuItem = (data, id) => {
+  let menuItem = createElem("li", {
+    id: "menu-item-" + id,
```

Comment:
Let's use [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)
```suggestion
    id: `menu-item-${id}`,
```

Reply:
fix